src/base/utils/misc.cpp

```diff
@@ -588,24 +589,11 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
     // QProcess::startDetached() explorer.exe still shows the default path. In
     // this case QProcess::startDetached() probably puts its own quotes around ours.
 
-    STARTUPINFO startupInfo;
-    ::ZeroMemory(&startupInfo, sizeof(startupInfo));
-    startupInfo.cb = sizeof(startupInfo);
-
-    PROCESS_INFORMATION processInfo;
-    ::ZeroMemory(&processInfo, sizeof(processInfo));
-
-    QString cmd = QString("explorer.exe /select,\"%1\"").arg(Utils::Fs::toNativePath(absolutePath));
-    LPWSTR lpCmd = new WCHAR[cmd.size() + 1];
-    cmd.toWCharArray(lpCmd);
-    lpCmd[cmd.size()] = 0;
-
-    bool ret = ::CreateProcessW(NULL, lpCmd, NULL, NULL, FALSE, 0, NULL, NULL, &startupInfo, &processInfo);
-    delete [] lpCmd;
-
-    if (ret) {
-        ::CloseHandle(processInfo.hProcess);
-        ::CloseHandle(processInfo.hThread);
+    CoInitializeEx(NULL, COINIT_MULTITHREADED);
+    ITEMIDLIST *pidl = ILCreateFromPathW(reinterpret_cast<PCTSTR>(Utils::Fs::toNativePath(path).utf16()));
+    if (pidl) {
+        SHOpenFolderAndSelectItems(pidl, 0, nullptr, 0);
+        ILFree(pidl);
     }
```

Comment:
>Call CoUnitialize at the end if the return of CoInitializeEx is S_OK or S_FALSE. (check remarks of CoInitializeEx)

Yes, I'm aware of this (sorry I didn't bring it up earlier). `CoInitializeEx()` is for initialize the COM library and (I thought) maybe it's better not to re-initialize & unload it every time when calling the function, thus I omitted the `CoUnitialize()`.
Any thoughts?

For point 2 & 4, I'll update it when we agree on point 1.

For `CoInitializeEx(NULL, COINIT_MULTITHREADED);`, because MSDN says:
>This parameter is reserved and must be NULL.